test/errorhandler.js

```diff
@@ -0,0 +1,36 @@
+// Copyright IBM Corp. 2015,2016. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+var app;
+var assert = require('assert');
+var request = require('supertest');
+var util = require('util');
+var async = require('async');
+var app = require('../');
+var loopback = require('loopback');
+var errorHandler = require('strong-error-handler');
+var boot = require('loopback-boot');
+
+describe('loopback.errorHandler(options)', function() {
+  it('should delete stack when options.debug is false', function(done) {
```

Comment:
You need two tests - one with `env: 'development'` and another with `env: 'production'`, they should be added to [test/end-to-end.js](https://github.com/strongloop/loopback-workspace/blob/cde85da8897425c0f4716e46c5bf9c95865df6a3/test/end-to-end.js#L35-L155) and `appRootDir` should be set to `SANDBOX`.